### PR TITLE
chore: update to final 1.9 driver

### PR DESCRIPTION
**Pull request by kblok**

### CODE REVIEW

File: src/PlaywrightSharp/PlaywrightSharp.csproj [hunk 1]
```diff
@@ -60,8 +60,6 @@
     </ContentWithTargetPath>
   </ItemGroup>
   <Target Name="EnsurePrerequisitsRan" BeforeTargets="InstallBrowsers">
-    <Error
-      Text="Playwright prerequisites are missing. Ensure you've ran `dotnet run -p ./src/tools/PlaywrightSharp.Tooling/PlaywrightSharp.Tooling.csproj -- download-drivers --basepath .`"
-      Condition="!Exists('$(MSBuildProjectDirectory)\Drivers\unix$(NativeFolder)')" />
+    <Error Text="Playwright prerequisites are missing. Ensure you've ran `dotnet run -p ./src/tools/PlaywrightSharp.Tooling/PlaywrightSharp.Tooling.csproj -- download-drivers --basepath .`" Condition="!Exists('$(MSBuildProjectDirectory)\Drivers\unix$(NativeFolder)')" />
```
Comment (avodovnik) [inline src/PlaywrightSharp/PlaywrightSharp.csproj:63]: I don't necessarily like this change - it reduces readability.

File: src/tools/PlaywrightSharp.Tooling/DriverDownloader.cs [hunk 2]
```diff
@@ -90,12 +91,7 @@ private static Task<string> GetUpstreamReadmeAsync(string playwrightVersion)
         private async Task DownloadDriverAsync(DirectoryInfo destinationDirectory, string driverVersion, string platform, string runtime)
         {
             Console.WriteLine("Downloading driver for " + platform);
-            string cdn = "https://playwright.azureedge.net/builds/driver";
-
-            if (driverVersion.Contains("next"))
-            {
-                cdn += "/next";
-            }
+            string cdn = "https://playwright.azureedge.net/builds/driver/next";
```
Comment (avodovnik) [inline src/tools/PlaywrightSharp.Tooling/DriverDownloader.cs:94]: Is this right, for a non-next version of `<DriverVersion>1.9.0-1614037901000</DriverVersion>`?

File: src/PlaywrightSharp/PlaywrightSharp.csproj (hunk 1 above)
Comment (kblok) [reply src/PlaywrightSharp/PlaywrightSharp.csproj:63]: yeah, IDE fight :(

File: src/tools/PlaywrightSharp.Tooling/DriverDownloader.cs (hunk 2 above)
Comment (kblok) [reply src/tools/PlaywrightSharp.Tooling/DriverDownloader.cs:94]: I took that from this change https://github.com/microsoft/playwright-python/pull/522/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L59

File: src/PlaywrightSharp/PlaywrightSharp.csproj (hunk 1 above)
Comment (avodovnik) [reply src/PlaywrightSharp/PlaywrightSharp.csproj:63]: I'll let you decide, but I'd discard it and rebase 👼🏻

File: src/tools/PlaywrightSharp.Tooling/DriverDownloader.cs (hunk 2 above)
Comment (avodovnik) [reply src/tools/PlaywrightSharp.Tooling/DriverDownloader.cs:94]: Hm, @aslushnikov can you clarify? Are we now supposed to use `/next` for all driver downloads?

Comment (avodovnik) [reply src/tools/PlaywrightSharp.Tooling/DriverDownloader.cs:94]: @kblok Andrey just confirmed this is right, so we're fine!

**Review by avodovnik — APPROVED** [review]

**Pull request merged**